app/styles/partials/nav-bar.scss

```diff
@@ -18,3 +18,16 @@
 .blue.text {
   color: #2185d0 !important;
 }
+
+.item.ui.dropdown {
```

Comment:
This will affect _all_ drop-downs across the site. Scope it to this one please. And increase the width of only the drop down and not the email/name item too.

Reply:
Done!.. scoped it to this element only and the tweaks from semantic classes are removed.